Traceback (most recent call last):
  File "/home/broug/Desktop/mlops-notebooks/02-src/preprocess_data_wandb.py", line 96, in <module>
    run_data_prep()
  File "/home/broug/mambaforge/envs/kaggle/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/broug/mambaforge/envs/kaggle/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/home/broug/mambaforge/envs/kaggle/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/broug/mambaforge/envs/kaggle/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/broug/Desktop/mlops-notebooks/02-src/preprocess_data_wandb.py", line 82, in run_data_prep
    os.makedirs(dest_path, exist_ok=True)
  File "<frozen os>", line 210, in makedirs
  File "<frozen posixpath>", line 103, in split
TypeError: expected str, bytes or os.PathLike object, not NoneType